Dockerfile

```diff
@@ -5,9 +5,13 @@ WORKDIR /usr/src/app
 
 COPY . .
 
-RUN apk add --no-cache --virtual build-dependencies gcc musl-dev\
-    &&  rm -rf /var/cache/apk/*
+# install build dependencies
+RUN apk add --no-cache --virtual build-dependencies gcc musl-dev
 
-RUN pip3 install --no-cache-dir -e .
+# install home-assistant-cli
+RUN pip3 install --no-cache-dir --editable .
+
+# remove build dependencies
+RUN apk del build-dependencies
```

Comment:
i'm not really following why you want this done - those files will be in the other layers. if it is to save space it should be done on the same line that creates them.

Reply:
ah i'm not sure why I wanted this too... fixed now which shrinked the image to around 110mb